### PR TITLE
Adding a new test case

### PR DESCRIPTION
**Pull request by omeneses**
go campi

### CODE REVIEW

File: src/main/java/pages/HomePageFS.java [hunk 1]
```diff
@@ -12,6 +12,17 @@ public GalleryOverlay goGalleryOverlay(){
         return new GalleryOverlay();
 
     }
+
+    public String getGalleryLinkText(){
+
```
Comment (cecampino) [inline src/main/java/pages/HomePageFS.java]: Remove empty lines

File: src/main/java/utils/CommonFuntions.java [hunk 2]
```diff
@@ -29,12 +29,14 @@ public static void createDriver (driverType type){
...
                 driver = new ChromeDriver();
+                break;
 
             case firefox:
                 System.setProperty("webdriver.gecko.driver","src/test/resources/drivers/geckodriver");
                 DesiredCapabilities capabilities = DesiredCapabilities.firefox();
                 capabilities.setCapability("marionette", true);
                 driver = new FirefoxDriver(capabilities);
+                break;
         }
```
Comment (cecampino) [inline src/main/java/utils/CommonFuntions.java]: Add default Statement

File: src/test/java/automation_workshop/GalleryTest.java [hunk 3]
```diff
@@ -18,7 +18,7 @@
     @Before
     public void before(){
 
-        createDriver(driverType.chrome);
+        createDriver(driverType.firefox);
```
Comment (cecampino) [inline src/test/java/automation_workshop/GalleryTest.java]: Why did we change chrome?

**Review by cecampino — CHANGES_REQUESTED** [review]
Please address changes before merging.

File: src/test/java/automation_workshop/GalleryTest.java (hunk 3 above)
Comment (omeneses) [reply src/test/java/automation_workshop/GalleryTest.java]: I just wanted use firefox  :)

File: src/main/java/utils/CommonFuntions.java (hunk 2 above)
Comment (omeneses) [reply src/main/java/utils/CommonFuntions.java]: Done

File: src/main/java/pages/HomePageFS.java (hunk 1 above)
Comment (omeneses) [reply src/main/java/pages/HomePageFS.java]: done

**Review by cecampino — APPROVED** [review]

**Comment by omeneses** [comment]
Adding branch_6, removing some empty lines and adding Default statement to switch cases

**Pull request merged**